kebechet/managers/update/update.py

```diff
@@ -400,7 +399,7 @@ def _open_merge_request_update(
         )
 
         _LOGGER.info("Creating a new pull request to update dependencies.")
-        merge_request = self.project.create_pr(
```

Comment:
There is another call in managers/version/version.py

Reply:
ack